pyproject.toml

```diff
@@ -1,14 +1,12 @@
 [tool.poetry]
 name = "poetry-plugin-export"
-version = "1.6.0"
+version = "1.8.0"
```

Comment:
why not 1.7.0?  there is no need to keep the plugin version synchronized with the poetry version

skipping a version will just cause people to raise issues asking why

Reply:
Changed it to 1.7.0